package.json

```diff
@@ -6,9 +6,10 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint --fix"
```

Comment:
Suggest you actually have both a `lint` script and a `lint:fix` script (or call them whatever you want. You want the linter to run in check mode on Github (so it just passes or fails a PR based on the linting), and you want the linter to run in fix mode locally (so your code is auto-fixed to match the linting rules before you push it to github).